index.html

```diff
@@ -543,6 +543,99 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem. It is a goal
+        of the v2 work to define a breaking change to the specification that
+        reduces production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they posses a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
+        </p>
+
+        <p>
+          If a member is not present in the <code>Credential</code> it MUST NOT
+          be present in the <code>VerifiableCredential</code> as either a claim
+          in the payload or a claim in the <code>vc</code> attribute of the
+          payload.
+        </p>
+
+        <p>
+          We start with an empty header, and payload objects, and we add members
+          to the header and the payload based on the content in the
+          <code>Credential</code>.
+        </p>
+
+        <section>
+          <h4>issuer -> iss</h4>
+
+          <p>
+            This member MUST be present in the
+            <code>payload.vc.issuer</code> attribute as either a string or an
+            <code>object</code> with and <code>id</code>.
+          </p>
+
+          <p>This member MUST be present <code>payload.iss</code>.</p>
+
+          <p>
+            In the case that <code>payload.vc.issuer</code> is an object,
+            <code>payload.iss</code> MUST be <code>payload.vc.issuer.id</code>.
+          </p>
+
+          <p>
+            In the case that
+            <code>payload.vc.issuer</code> is a string,
+            <code>payload.iss</code> must be
+            <code>payload.vc.issuer</code>
+          </p>
```

Comment:
why define this logic if it is not a top-level claim `iss`?

Reply:
because you may be mapping from:

```
issuer: { id, name } -> payload.iss === id
```

and you don't want someone to do this:

```
issuer: { id, name } -> payload.iss === { id, name }
```

^ AFAIK, this is illegal regarding iss:
 
 
 >  The "iss" (issuer) claim identifies the principal that issued the
 >   JWT.  The processing of this claim is generally application specific.
 >   The "iss" value is a case-sensitive string containing a StringOrURI
 >   value.  Use of this claim is OPTIONAL.
   
 https://www.rfc-editor.org/rfc/rfc7519.html#section-4.1.1

Reply:
I think `principal` (at least as used in JWT specs) is closer to `agent` or `entity` than it is to `subject`.

Reply:
interesting... @selfissued @Sakurann @tplooker what do ya'll think?

Reply:
Yes, "principle" in the JWT spec is a generic term like "entity".  We use it in the description of the "iss", "sub", and "aud" claims.  I don't think its use would be particularly useful in this spec, unless perhaps in usage parallel to that in the JWT spec.

Reply:
I think there is a general mapping between `entity` and `issuer`, `holder` and `verifier`... thats probably best discussed on issues, please continue discussion here: https://github.com/w3c/vc-jwt/issues/35